packages/KaSim/KaSim.3.5.141112/url

```diff
@@ -1,2 +0,0 @@
-http: "https://github.com/Kappa-Dev/KaSim/archive/v3.5-121114.tar.gz"
-checksum: "4df6a6b855e8cf070a2e418749b22d2c"
```

Comment:
Would it be possible to keep the older version, unless it is completely broken?  We prefer to keep historical versions and let users choose when to upgrade.